src/rf/js/src/home/components/library.js

```diff
@@ -543,7 +554,7 @@ var ImageMetadata = React.createBackboneClass({
                 <div className="layer-detail-content">
                     <img className="img-preview" src={image.thumb_large || 'https://placehold.it/300x300'} />
                     <hr />
-                    <div className="scrollable">
+                    <div className="scrollable" style={{overflow: 'hidden'}}>
```

Comment:
Do we want overflow hidden or just in one direction. I had trouble with the overflow being hidden if there was more metadata than would fit vertically in the panel.